tests/test_bad_eventlog.py

```diff
@@ -3,46 +3,41 @@
 from pathlib import Path
 from zipfile import ZipFile
 
-from spark_log_parser.eventlog import EventLogBuilder
+from spark_log_parser import eventlog, extractor
 
 
 class BadEventLog(unittest.TestCase):
-    def test_multiple_context_ids(self):
-        event_log = Path("tests", "logs", "bad", "non-unique-context-id.zip").resolve()
+    def check_value_error(self, event_log_path, msg):
 
         with tempfile.TemporaryDirectory() as temp_dir:
-            with self.assertRaises(
-                ValueError, msg="Not all rollover files have the same Spark context ID"
-            ):
-                EventLogBuilder(event_log.as_uri(), temp_dir).build()
+            with self.assertRaises(ValueError) as cm:
+
+                event_log_paths = extractor.Extractor(event_log_path.as_uri(), temp_dir).extract()
+                eventlog.EventLogBuilder(event_log_paths, temp_dir).build()
+
+            assert str(cm.exception) == msg  # , "Exception message matches"
```

Comment:
Just curious - why drop the message?

Reply:
Oops that was an oversight. Neal also suggested switching the messages to a negative phrasing ("eventlog missing" vs "eventlog present") since its more accurate. I did that throughout.